aim/cli/convert/processors/mlflow.py

```diff
@@ -41,11 +41,17 @@ def parse_mlflow_logs(repo_inst, tracking_uri, experiment):
 
     client = mlflow.tracking.client.MlflowClient(tracking_uri=tracking_uri)
 
+    experiment = kwargs.get('experiment_id') or kwargs.get('experiment_name')
+    get_experiment_by_name = kwargs.get('experiment_name') == experiment
+
     if experiment is None:
         # process all experiments
         experiments = client.list_experiments()
     else:
-        experiments = (client.get_experiment_by_name(experiment),)
+        if get_experiment_by_name:
```

Comment:
lets use the single `--experiment` option.
try with name first, if experiment is not found try treat it as experiment ID
if the experiment is not found, echo an error message and exit

Reply:
well yeah good idea